### DISCUSSION TITLE
The scan QR code function has stopped working for me

### DISCUSSION THREAD

**Discussion by Heros-Tempus** [2fast-team/2fast#144] [Q&A] [answered]
In the past I was able to add accounts using the scan QR code function without any issue. But when I tried to add a new account via QR code today, the app did not behave as expected and I was not able to scan the QR code. Instead of counting down before connecting to an instance of the snipping tool for scanning the QR code, it would just instantly open the snipping tool app in its own window. Using that window to scan the QR code did nothing but create an image that I could save to my computer. I don't know if it's the 2fast app, the snipping tool app, or something else on my laptop that is causing the issue. But something has changed and is disrupting 2fast's ability to connect to the snipping tool on my laptop.

**Comment by jp-weber** [comment, marked as answer]
The issue is related to a change by Microsoft. As workaround you can reset this configuration: see https://github.com/2fast-team/2fast/issues/133#issuecomment-2094891298

A new version will be released in a few days, a beta version already delete the dependency on the snipping tool: https://github.com/2fast-team/2fast/releases/tag/v1.3.0-beta
